Assets/Scripts/UI/Overlay/OverlayMap.cs

```diff
@@ -287,10 +270,11 @@ public void SetOverlay(string name)
                 }
 
                 Tile tile = WorldController.Instance.GetTileAtWorldCoord(new Vector3(x, y, z));
-                return (int)script.Call(handle, new object[] { tile, World.Current }).ToScalar().CastToNumber();
+                DynValue result = FunctionsManager.Overlay.Call(descr.LuaFunctionName, new object[] { tile, World.Current });
+                return (int)result.ToScalar().CastToNumber();
```

Comment:
If result is null or non-numeric the resulting exception wouldn't easily trace back to the guilty lua function, or would it? (I'm expecting it would be "chunk blah blah blah" or a random NRE or the like.) Might just want to do a check here and throw an error msg with some meaningful information?